README.md

```diff
@@ -21,32 +39,33 @@ $ psql -f db/schema.sql -d soapbox_dev
 
 2. **Build and install the Soapbox API server (soapboxd) and CLI client (soapboxcli):**
 ``` shell
+$ mkdir -p $GOPATH/src/github.com/adhocteam
+$ ln -s /path/to/soapbox $GOPATH/src/github.com/adhocteam/soapbox
```

Comment:
This is not recommended. The recommend way is to clone the repo directly to its package's path in the context of GOPATH.

`go get github.com/adhocteam/soapbox/...` will clone the repo and place it in the right location, and should be sufficient.